.github/workflows/helm-release.yaml

```diff
@@ -56,19 +57,19 @@ jobs:
           if [[ ${{ github.event_name }} == "push" ]]; then
             args="${args} --dev"
           fi
-
-          ./tools/releases/helm.sh ${args}
-
+          ${{ env.KUMA_DIR }}/tools/releases/helm.sh ${args}
+      - name: Commit chart versions
+        run: |
           git add -u deployments/charts
           # This commit never ends up in the repo
-          git commit -m "ci(helm): update versions"
+          git commit --allow-empty -m "ci(helm): update versions"
```

Comment:
Why do we need to `--allow-empty`? I vaguely remember some conversation about this on slack and I'm not sure if this does not have any negative ramifications (or maybe it was dirty index and adding `-am`).

Reply:
KM has allow-empty and this didn't. Had to pick one and picked adding `--allow-empty` as mentioned this commit is not actually pushed anywhere so I think it doesn't matter (and avoids failing git commit)